source/modulo_core/src/Component.cpp

```diff
@@ -17,10 +17,6 @@ Component::Component(const rclcpp::NodeOptions& options) : Cell(options) {
   this->on_init();
 }
 
-Component::~Component() {
-  this->on_shutdown();
```

Comment:
Thanks for removing this. It prevents multiple invocations of on_shutdown (which wasn't necessarily doing any harm since on_shutdown used to be empty, but just printing "on shutdown of the Cell class called" twice, which was always annoying)